modules/oneVideoBidAdapter.js

```diff
@@ -165,14 +165,17 @@ function getRequestData(bid, consentData, bidRequest) {
   let loc = bidRequest.refererInfo.referer;
   let page = (bid.params.site && bid.params.site.page) ? (bid.params.site.page) : (loc.href);
   let ref = (bid.params.site && bid.params.site.referrer) ? bid.params.site.referrer : bidRequest.refererInfo.referer;
+  let getFloorRequestObject = {
+    currency: bid.params.cur || 'USD',
+    mediaType: 'video',
+    size: '\*'
```

Comment:
is the '\' character here a typo? haven't seen that in other implementations

Reply:
it was taken from the adapter example in https://docs.prebid.org/dev-docs/modules/floors.html#bid-adapter-interface
originally i thought there was an escaped character issue with the string once it's passed to .getFloor()
I can easily remove and test again.

Reply:
removed backslash

Reply:
@bszekely1 is there a typo in the doc? interestingly doesn't seem to effect things

Reply:
That is a typo. Let me fix it. I would recommend to remove it even if it works.

Reply:
Updated. Thanks for catching it.